reference/5-api-publisher/API-Connection-Management.md

```diff
@@ -0,0 +1,64 @@
+# API Connection Management
+
+The Quick Start exercise supplies all connection information explicitly as command-line arguments, however to better operationalize the API Publisher you should create a Configuration Store to securely configure named API connections, and to track the state of publishing between various source and target APIs.
+
+The Configuration Store implementation can be selected by passing the `--configurationStoreProvider` command-line argument, by setting the `EdFi:ApiPublisher:ConfigurationStore:Provider` environment variable, by modifying the `provider` value in the _configurationStoreSettings.json_ file (in that order of precedence).
+
```

Comment:
Please wrap all long lines at 80 characters. I find the [rewrap extension](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) very useful for this. After installing the extension, use <kbd>ALT+Q</kbd> to rewrap a paragraph. You can select all text and rewrap the entire file with that one command.

See #22 for other extensions that I find useful. With the `markdownlint` extension installed, you can automate fixing a lot of the little problems mentioned below.

After installing the extension, choose `Format Document with...` from the Visual Studio command Palette.

Reply:
fixed